download_data.py

```diff
@@ -35,12 +35,13 @@ def download_data(
     if isinstance(dataset, datasets.arrow_dataset.Dataset):
         filename = dataset_subset + ".parquet"
         dataset.to_parquet(dataset_dir / filename)
+        filename = args.dataset_subset + ".parquet"
```

Comment:
This is redundant, the only difference adding in "args." Did you mean to delete the previous lines, or is this intentional?